src/xcode/ENA/ENA/Source/Services/PPAnalyticsCollector/Model/PPADataType.swift

```diff
@@ -13,6 +13,25 @@ enum PPADataType {
 	case keySubmissionMetadata(PPAKeySubmissionMetadata)
 	case exposureWindowsMetadata(PPAExposureWindowsMetadata)
 	case submissionMetadata(PPASubmissionMetadata)
+	
+	var description: String {
+		switch self {
+		case .userData:
+			return "userData"
+		case .riskExposureMetadata:
+			return "riskExposureMetadata"
+		case .clientMetadata:
+			return "clientMetadata"
```

Comment:
I don't see this case exists in the enum

Reply:
@KaiTeuber  I updated the code to 2.1 and it is now removed